test/api-test/api/metadata/jobcodes/{id}/update/post-pre-request.js

```diff
@@ -0,0 +1,20 @@
+let jobCodes = [
+  {
+    "code": "95-1240",
+    "targetNodeName": "Name 20230622-1519",
+    "frameworkName": "Framework 20230622-1519",
+    "jobCodeLevelAsNumber": 3
+  }
+];
+
+let body = {
+  mode: 'raw',
+  raw: JSON.stringify(jobCodes),
+  options: {
+    raw: {
+      language: 'json'
+    }
+  }
+};
+
+pm.request.body.update(body);
```

Comment:
You should either pass a new id that you are updating "id": 2520 here in this file. Or change the id in the post.json file to "21" because that is what is going to be in the path.